common/background-image.c

```diff
@@ -21,6 +24,103 @@ enum background_mode parse_background_mode(const char *mode) {
 	return BACKGROUND_MODE_INVALID;
 }
 
+#if HAVE_GDK_PIXBUF
+static cairo_surface_t* gdk_cairo_image_surface_create_from_pixbuf(
```

Comment:
Is it intentional to move this from `cairo.c` to this file?

Reply:
Yeah.